chef/site-cookbooks/wca/recipes/default.rb

```diff
@@ -103,7 +103,7 @@
   action :create_if_missing
 end
 
-node.default['brightbox-ruby']['version'] = "2.3"
+node.default['brightbox-ruby']['version'] = "2.4"
```

Comment:
Would it make sense to "DRY" this up like this?

`node.default['brightbox-ruby']['version'] = RUBY_VERSION`